python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -26,10 +26,12 @@ from cuml.common.base import Base
 from cuml.metrics.accuracy import accuracy_score
 from cuml.utils import input_to_dev_array
 from cuml.utils.cupy_utils import checked_cupy_unique
+import rmm
+import numpy as np
 
 supported_penalties = ['l1', 'l2', 'none', 'elasticnet']
 
-supported_solvers = ['qn', 'lbfgs', 'owl']
```

Comment:
Small question: shouldn't we deprecate `lbfgs` and `owl` for 0.12 to not just break in the (probably small chance of ) case of someone currently using? It probably won't affect many users, but this could be a good chance to use deprecation practices

Reply:
Interesting... my contention is that this has always been a bug. Choosing LBFGS or OWL as the solver has no effect in logistic regression (:man_facepalming:) so anyone using them has been getting silently ripped off in the past.

Reply:
yeah I think this one is fine since owl in particular is there for L1 so it is automatically selected, this design for the supported solvers was just while things were getting sorted out so dropping it probably will have no consequences